app/data/census_household.json

```diff
@@ -1163,6 +1194,13 @@
                                 }
                             ]
                         }
+                    ],
+                    "routing_rules": [
+                        {
+                            "goto": {
+                                "id": "household-member-completed"
```

Comment:
Yeah, don't see the point of adding a routing rule that is going to break, especially if we're modifying navigator in order to make it jump outside of a group.